meterfs/common.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Phoenix-RTOS
+ *
+ * Meterfs tests common part
+ *
+ * Copyright 2021 Phoenix Systems
+ * Author: Tomasz Korniluk
+ *
+ *
+ * %LICENSE%
+ */
+
+#include "common.h"
+
+void common_fileInfoFill(file_info_t *info, size_t sectors, size_t filesz, size_t recordsz, size_t recordcnt)
+{
+	info->filesz = filesz;
+	info->sectors = sectors;
+	info->recordsz = recordsz;
+	info->recordcnt = recordcnt;
+}
+
+
+void common_preallocOpenFile(const char *name, size_t sectors, size_t filesz, size_t recordsz, int *fd)
+{
+	char fileName[9];
+
+	TEST_ASSERT_EQUAL(0, file_allocate(name, sectors, filesz, recordsz));
+	sprintf(fileName, "/%s", name);
+	*fd = file_open(fileName);
+	TEST_ASSERT_GREATER_OR_EQUAL(0, *fd);
+}
+
+
+void common_readContent(int fd, size_t offset, void *buff, size_t bufflen, void *content, int contentsz, const char *msg)
+{
+	int i;
+	char tempBuff[bufflen];
```

Comment:
Are the `VLA` allowed?

Reply:
I corrected that. Now it uses global buff in common struct.

Reply:
I corrected that. Now it uses global buff in common struct.